docs/content/connectors/postgres-cdc.md

```diff
@@ -48,6 +48,7 @@ CREATE TABLE shipments (
   'database-name' = 'postgres',
   'schema-name' = 'public',
   'table-name' = 'shipments',
+  'slot.name' = 'flink',
    -- experimental feature: incremental snapshot (default off)
```

Comment:
Please update the sql at line 367 too.

Reply:
i have update the sql at line 367 with commint b915ec49a572a2cd3cfe6fde09f02942f60cf4b5

Reply:
![image](https://github.com/ververica/flink-cdc-connectors/assets/46524102/fe3f2d52-8693-410b-a824-c7c24641de82)
as u can see the code for two differecnt line

Reply:
@ruanhang1993  if u have free time ,please review it